events/Glue/UpdateJob.json

```diff
@@ -9,6 +9,35 @@
     "mitreAttackTechniques": [
         "T1098 - Account Manipulation"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1036 - Masquerading",
+            "reason": "Adversaries can modify the job definition to make the job appear legitimate, effectively hiding malicious activities within a seemingly benign job."
+        },
+        {
+            "technique": "T1070 - Indicator Removal",
+            "reason": "Updating a job definition can include instructions to remove or alter logs and other artifacts, helping adversaries evade detection."
+        },
+        {
+            "technique": "T1484 - Domain or Tenant Policy Modification",
+            "reason": "Adversaries can change job definitions to alter the behavior of the Glue job, affecting the overall data processing policies and workflows within the AWS environment."
```

Comment:
This technique seems to much related with AD or Trust policies between domain, for this reason I'm not sure this one matches here.

Reply:
I am also a byte unsure. I will remove it for now.